### DISCUSSION TITLE
Repository signing for extension file failed. Please retry the submission or contact ...

### DISCUSSION THREAD

**Discussion by sketchbuch** [microsoft/vscode-discussions#1084] [Extension Development QnA]
I was updating an extension just now and verification took ages. I eventually get the error that:

> Repository signing for extension file failed. Please retry the submission or contact VSMarketplace@microsoft.com.

Is there any way to view some kind of bulld log to find out why?

![image](https://github.com/microsoft/vscode-discussions/assets/16445037/e88568b5-6c4c-444e-9d84-1a3221edad87)

[I just setup some github actions](https://github.com/sketchbuch/vsc-workspace-sidebar/blob/master/.github/workflows/publish.yml) to publish my extension to the marketplace, could this be why? I also created a new token. Maybe it would have worked if I used the token I used to use for manual publishing?

The Open VSX publication seems to be ok, at least it seems to have been published (v2.0.0)

**Comment by sketchbuch** [comment]
Manual publishing worked v2.0.2, but it took ages - 12 minutes.

**Reply by burkeholland** [reply]
Is the perf still degraded for verification? I'm wondering if it was a point in time issue.

**Comment by sketchbuch** [comment]
It seems to be fixed now, publishing was quick.
